roles/jenkins/vars/main.yml

```diff
@@ -1,10 +1,17 @@
 jenkins_version: '2.89.1'
+this_repository: 'https://github.com/jakobalander/ansible-jenkins.git'
 disable_setup_wizard: '-Djenkins.install.runSetupWizard=false'
 jenkins_admin_user: "{{ vault_jenkins_admin_user }}"
 jenkins_admin_pwd: "{{ vault_jenkins_admin_pwd }}"
+jenkins_dir: /var/lib/jenkins
+jenkins_user: jenkins
+jenkins_group: jenkins
+jenkins_seed_job:
+  name: ~seed_job
+  dsl_script_path: '**/generate_jobs_jenkinsfile.groovy'
 
 plugins:
```

Comment:
The list of plugin quickly becomes unmaintainable and might contain duplicates if not sorted. There is a line sort plugin for intellij, just install it and sort the plugins alphabetically.